src/06-errors_and_files/04-implementation.md

```diff
@@ -4,9 +4,8 @@ This was a long info dump. Let's practice what we've learned. We want to:
 
 - Create the output directory
 - Grab all file names in a directory
-- Filter them according to their extension, we want to process `txt` file and
-- copy other files without modification
-- We want to parse each text file, build an index of the result,
+- Filter them according to their extension, we want to process `txt` files and copy other files without modification
```

Comment:
the idea here is to separate these into steps where each bullet point is a step we need to implement, do you have a suggestion how to format it so it works?

Reply:
Maybe have three steps:
- Filter them according to their extension
- Process `.txt` files
- Copy other files without modification

Reply:
Yeah that sounds good to me. I'll add a suggestion:

```suggestion
- Filter them according to their extension
- Process .txt files
- Copy other files without modification
```